redash/handlers/queries.py

```diff
@@ -203,6 +203,13 @@ def get(self):
             with_last_modified_by=False
         )
 
+        if search_term:
+            self.record_event({
+                'action': 'search',
+                'object_type': 'query',
+                'term': search_term,
+            })
```

Comment:
A missing event recording in #2766 for the new hybrid query view that also can be used for search.

Reply:
👍 
If we're at it, then also need to record a list event when it's not search. Also worth recording other parameters of the request - can be useful to analyze if people use tag based navigation or how deep they go.

Reply:
Good idea!

Reply:
Added.